### PR TITLE
add test to Subjects component #121

### LINKED ISSUE #121: (SP: 1) Write unit test for "Subjects" page
### Page unit test
Unit test for "**Subjects**" page
Scenaries descriptions:
- [x] It should render page
- [x] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/pages/subjects/Subjects.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by Mygal-Alex**
image

### CODE REVIEW

**Review by AndriiNasadiuk — APPROVED** [review]
you can ommit beforeEach(), if you have 1 it() statement

**Review by marianamartsy — APPROVED** [review]
ok
ps. beforeEach don't needed in this case, I guess

**Review by LeonidShamarin — APPROVED** [review]

**Review by ArtemHolikov — APPROVED** [review]
What can be easier?) Good job. But can we render Subjects inside functional block? Why it`s better?

File: tests/unit/pages/subjects/Subjects.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/pages/subjects/Subjects.spec.jsx:1]: Mock `PageWrapper` and check 2 conditions (two  `expect` or two `it`) according to the task.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**